### ISSUE TITLE
Support for Senisbo Pure

### ISSUE THREAD

**Issue by koshisan** [thetonto/node-red-contrib-sensibo#22]
Hi!
I just received two Sensibo Pure units and was looking into smarthome integration. I stumbled upon your node-red node - however, it seems to be build with only AC in mind...

Would you be willing to integrate support for the Pure devices?

They are using the same API and mostly the same fields - to the level where they even get detected right now. It is just that some fields would need to be adjusted.

For instance there are three additional measurements: pm25, tvoc and co2. temperature and humidity are still present, but always zero.

Control is also done under acState. It still has the settings for "on" and "fanLevel". fanLevel only knows "low" and "high".

There is an additional setting called "light" which can be on, off or dim

So basically it shouldn't be to hard.
